packages/ui/src/components/icon-v2/icon-v2.tsx

```diff
@@ -35,6 +35,11 @@ const IconV2: FC<IconPropsV2> = ({
       ? IconNameMapV2[`${name}-light` as keyof typeof IconNameMapV2]
       : IconNameMapV2[name]
 
+  if (!Component) {
+    console.warn(`Icon "${name}" not found in IconNameMapV2.`)
```

Comment:
If you pass the wrong icon name when making a doc page in the Portal app, it crashes, and there are no TS suggestions, which could prevent it. This fixes it.